health-services/project-factory/src/server/utils/genericUtils.ts

```
@@ -280,6 +278,55 @@ async function searchGeneratedResources(request: any) {
   }
 }
 
+async function searchAllGeneratedResources(searchQuery: any, locale: any) {
+  try {
+    const { type, tenantId, hierarchyType, id, status, campaignId } = searchQuery;
+    let queryString = `SELECT * FROM ${config?.DB_CONFIG.DB_GENERATED_RESOURCE_DETAILS_TABLE_NAME} WHERE `;
+    let queryConditions: string[] = [];
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'queryConditions' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)